src/helpers/passable_args/index.js

```diff
@@ -17,27 +17,51 @@
  * @param {Array.<{specific: String[], passes: passableCallback, custom: Object}>} args - arguments for Passable configuration
  * @return {object} Passable configuration settings
  */
-function passableArgs(args: PassableArguments) {
+function passableArgs(args: PassableArguments): PassableRuntime {
 
-    let passes: Function,
-        specific: specific = [],
+    let passes: Passes,
+        specific: Specific = [],
         custom: Rules = {};
 
-    switch (args.length) {
-      case 0:
-        throw new TypeError('[passable]: Failed to execute `passableArgs`: At least 1 argument required, but only 0 present.');
+    let res: [Array<string> | string, Passes, Rules];
 
-      case 1: // [passes] = args;
-        if (typeof args[0] != 'function') {
-          throw new TypeError('[passable]: Failed to execute `passableArgs`: Unexpected ' + typeof args[0] + ', expected function');
-        }
-        args = [specific, ...args];
+    switch (args.length) {
+        case 0:
```

Comment:
It's super small and petty, but I think it would look better if you capitalize `passable` in the error messages, same as you did in `types`.
Also, some strings use single quotes, and some double.